framework/templates/ecl-templates-search/ecl-templates-search.config.js

```diff
@@ -1,12 +1,16 @@
 module.exports = {
   title: 'Search Page',
   label: 'Search Page',
-  status: 'wip',
+  status: 'ready',
   preview: '@preview-full-page',
   tags: ['template'],
   context: {
     global: {
       language: 'en',
     },
+    _demo: {
+      scripts:
```

Comment:
`Hide`, `Refine`... the default `expandable` behavior doesn't allow to change the label of the button. We could add this feature (using `data-` attributes for example) if it makes sense.

Regarding the form, addind the `.ecl-u-d-lg-block` class on it forces it to be displayed on large viewport (bypassing the `aria-hidden`) :)